MdePkg/Library/BaseIoLibIntrinsic/BaseIoLibIntrinsicTdx.inf

```diff
@@ -40,7 +40,7 @@
   IoLibMscTdx.c    | MSFT
   IoLibInternalTdx.c
   IoLibTdx.c
-  X64/IoFifoSev.nasm
+  X64/IoFifo.nasm
```

Comment:
> Can you explain why use X64/IoFifo.nasm instead of X64/IoFifoSev.nasm?
> IoFifoSev.nasm supports AMD SEV features and BaseIoLibIntrinsicTdx is supposed to support TDX/SEV/Normal Ovmf IO operations.

My understanding is not correct. The change is not needed. I will update.